### DISCUSSION TITLE
Can't resolve imported dependency "quasar/lang/pt-br"

### DISCUSSION THREAD

**Discussion by murilolivorato** [quasarframework/quasar#12097] [CLI - SSR mode] [answered]
hello, I have a project that I start in a computer , after I changed of machine , download the project from git , and started the project again ( in other machine ) .
I did -

* npm install
* I installed Quaser Cli - npm install -g @quasar/cli
* After I did - npm quasar dev

It gave this error -
```
 App •  ERROR  •  UI  in ./.quasar/quasar-user-options.js

Module not found: Can't resolve imported dependency "quasar/lang/pt-br"
Did you forget to install it? You can run: npm install --save quasar/lang/pt-br


 App •  COMPILATION FAILED  • Please check the log above for details.

```
in my conf -
```
lang: 'pt-br', // Quasar language pack

```

I am using linux , and before I was using windowls
I am using quasar - 2.0 
It is look link that it didnt install the language package when I did the npm install

**Comment by MilosPaunovic** [comment]
Seems like you haven't imported your file from proper path, so that should be the first thing to check.

**Comment by pdanpdan** [comment]
If using Quasar v2 the last part of the locale id should be uppercase: pt-BR

**Comment by rstoenescu** [comment, marked as answer]
Windows has a case insensitive filesystem, while any other OS has a case-sensitive one.
This is why although you've mistakenly set it to "pt-br", it did work. However, the issue was surfaced when you moved to Linux. Change it to the right "pt-BR" and it'll work.

**Comment by murilolivorato** [comment]
Thanks @pdanpdan, @MilosPaunovic, @rstoenescu .
It worked .
